### PR TITLE
Update plugin for linting and coveralls

### PR DESCRIPTION
**Pull request by rachelnicole**
Update package.json / eslint / coveralls.

### CODE REVIEW

**Comment by Snugug** [comment]
Please include validation to test to ensure that the input is a valid URL, removing the "empty" validation, and update tests accordingly

File: tests/validation.js [hunk 1]
```diff
@@ -1,37 +1,19 @@
...
 });
 
-// Invalid input
-test('validate correct input', t => {
-  const ip = input;
-  ip.target.value = '';
-
-  t.is(validation(ip, settings), 'url cannot be left blank!', 'Return string if not valid');
+test('validate text is url', t => {
+  t.true(validation(input), 'Must be an actual url');
```
Comment (scottnath) [inline tests/validation.js:25]: @rachelnicole please write a failing test. When you look at your code coverage:
https://travis-ci.org/punchcard-cms/input-plugin-url/builds/132883073
You can see in the nyc-coverage chart that line number 26 is not tested:

```
-----------------------|----------|----------|----------|----------|----------------|
File                   |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
-----------------------|----------|----------|----------|----------|----------------|
 input-plugin-url/     |      100 |      100 |      100 |      100 |                |
  index.js             |      100 |      100 |      100 |      100 |                |
 input-plugin-url/lib/ |       80 |       50 |      100 |       80 |                |
  validation.js        |       80 |       50 |      100 |       80 |             26 |
-----------------------|----------|----------|----------|----------|----------------|
All files              |    85.71 |       50 |      100 |    85.71 |                |
-----------------------|----------|----------|----------|----------|----------------|
```

line 26 is this line:

```
    return 'Not a valid url.';
```

so we need a test to check strings which are not urls, which will then ensure that the validation works as expected for both valid and invalid urls.

**Comment by rachelnicole** [comment]
@scottnath test added for invalid url check!

selfie-0
[_GitHub Selfies_](https://github.com/thieman/github-selfies/)

File: tests/validation.js [hunk 2]
```diff
@@ -1,37 +1,30 @@
...
 
+
 const input = {
   target: {
     name: 'url',
-    value: 'foo bar baz',
-  },
-  all: {
-    url: 'foo bar baz',
+    value: 'ibm.com',
```
Comment (scottnath) [inline tests/validation.js]: Can we please use a more generic url?
